src/desktop/components/inquiry_questionnaire/map/decisions.coffee

```diff
@@ -5,7 +5,7 @@ hasSeenThisSession = (steps...) -> ({ logger }) -> logger.hasLoggedThisSession s
 
 decisions =
   ask_specialist: ({ artwork, ask_specialist }) ->
-    artwork.get('is_in_auction') is true || ask_specialist is true
+    artwork.get('is_in_auction') is true || ask_specialist is true || artwork.get('ecommerce') is true
```

Comment:
Here is where we seem to decide what dialog to display. Not sure if I should have used the same `is_acquireable` logic here but `ecommerce` seem to be properly set on artwork at that point.

Reply:
Could also do `artwork.get('is_in_auction') || artwork.get('ecommerce') || ask_specialist`